tools/platformio-build.py

```diff
@@ -106,13 +106,6 @@ def is_pio_build():
         "ARDUINO_ARCH_RP2040",
         ("F_CPU", "$BOARD_F_CPU"),
         ("BOARD_NAME", '\\"%s\\"' % env.subst("$BOARD")),
-        # LWIP-related
-        ("PICO_CYW43_ARCH_THREADSAFE_BACKGROUND", 1),
-        ("CYW43_LWIP", 0),
-        ("LWIP_IPV6", 1),
-        ("LWIP_IPV4", 1),
-        ("LWIP_IGMP", 1),
-        ("LWIP_LWIP_CHECKSUM_CTRL_PER_NETIF", 1),
```

Comment:
Ahh, now I know why the original builder script didn't work, there was a double `LWIP_LWIP` here. Good catch, I accidentally duplicated them in my PR then, but you removed them here so that it can happen in `configure_network_flags()`.